### DISCUSSION TITLE
Name changes in Butterfly

### DISCUSSION THREAD

**Discussion by claws99** [LinwoodDev/Butterfly#479] [Ideas]
Pull request opened at https://github.com/LinwoodDev/Butterfly/pull/486

---

The current (until 2.00 beta 6)

* Hand  to be  renamed to `Select` 
* Painters will be renamed to Tools
* Export>Document instead of Export>Data
* Packs to something else
* Label to Text
* Move to Hand

**Comment by CodeDoctorDE** [comment]
* Rename BoxBackground to PatternBackground
* Rename TextElement to RichTextElement

**Comment by CodeDoctorDE** [comment]
Hi, I wouldn't call the label tool text tool.
In the code the label tool can create markdown and text elements which conflicts with the name.
Do you remember which name we agreed on for packs?

**Comment by CodeDoctorDE** [comment]
File version was bumped to 10
